src/Node/Buffer/ST.purs

```diff
@@ -74,10 +83,23 @@ instance mutableBufferST :: MutableBuffer (STBuffer h) (ST h) where
   copy = copy
   fill = fill
 
+-- | Creates a new `STBuffer`. Alias of `alloc`.
 create :: forall h. Int -> ST h (STBuffer h)
-create = unsafeCoerce Buffer.create
```

Comment:
Are these exact replacements, in that someone who was previously using `create` will not be affected by a change to `alloc` underneath?

Reply:
The [original FFI](https://github.com/purescript-node/purescript-node-buffer/blob/v8.0.0/src/Node/Buffer/Immutable.js#L17-L19) is using `alloc` underneath. However, from the name alone, it's not clear which Node API is being used. 

I made `create` an alias of `alloc` to reduce breakage. I think I'd prefer to remove it entirely, but that can be discussed in a separate PR as there are other cases like that.

Reply:
I see. Thanks for clarifying.